### PR TITLE
feat: create study with a simple config

### PR DESCRIPTION
**Pull request by sssoja**
# Changelog

- created a config mocks folder to store real study data
- implemented a `simple` controller that takes a config of type `configObject` and returns a set of fields
- developed a `formBuilder` that takes a single field and returns a translated version configured for rendering a form
- unit tested output across all modules and helpers 
- separated the concept of creating a new study and creating a new study conf

Fixes: #82

### CODE REVIEW

File: dashboard-frontend/mocks/configs/fly-bulgaria.json [hunk 1]
```diff
@@ -0,0 +1,1050 @@
...
+      {
+        "initial_shortcode": "bebbobgbasebul",
+        "name": "fly"
+      }
+    ]
+  },
+  {
+    "conf_type": "general",
+    "conf": {
+      "ad_account": "1342820622846299",
```
Comment (Spazzy757) [inline dashboard-frontend/mocks/configs/fly-bulgaria.json:74]: @sssoja this has some sensitive data in it, we should clean out these mocks

File: dashboard-frontend/src/pages/NewStudyPage/form/configs/general.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
+export const general = {
+  type: 'configObject',
+  title: 'General',
+  description: 'The "general" configuration consists of... General stuff?',
```
Comment (Spazzy757) [inline dashboard-frontend/src/pages/NewStudyPage/form/configs/general.js:4]: maybe we update this description to point o a document that has indepth explanation of what exactly this is (could just be our github for now)

**Comment by Spazzy757** [comment]
We need to finalize this PR: https://github.com/vlab-research/vlab/pull/120 before we can look at merging the Simple Config. Also potentially we need to add the API calls to this PR

File: dashboard-frontend/mocks/configs/fly-bulgaria.json (hunk 1 above)
Comment (Spazzy757) [reply dashboard-frontend/mocks/configs/fly-bulgaria.json:74]: or just with "Fake" data

**Comment by sssoja** [comment]
Closing PR to work on feat: create study with a general conf #127

**Pull request closed without merging**